app/services/katello/candlepin/consumer.rb

```diff
@@ -137,25 +137,26 @@ def self.distribution_to_puppet_os(name)
         return ::Operatingsystem::REDHAT_ATOMIC_HOST_OS if name == ::Operatingsystem::REDHAT_ATOMIC_HOST_DISTRO_NAME
 
         name = name.downcase
-        if name =~ /red\s*hat/
+        case name
```

Comment:
We could simplify this to `case name.downcase` now.

Reply:
Good catch. Thanks

Reply:
@m-bucher i've updated the changes, can you take a look?